pandas/core/strings.py

```diff
@@ -1180,19 +1180,36 @@ def str_slice(arr, start=None, stop=None, step=None):
 
 def str_slice_replace(arr, start=None, stop=None, repl=None):
     """
+    Replace a sliced string.
```

Comment:
Maybe it's implied, but I'd like to see this say what the sliced bit is replaced with

    Replace a sliced string with another value.

    Like all of pandas string methods, this operates elementwise on each string
    in the Series or Index.

and then the rest of the docstring.

Reply:
I think "a slice of a string" is easier to understand than "sliced string"